packages/dropdown/src/Dropdown.js

```diff
@@ -14,67 +14,80 @@ import renderOptions from "./presenters/renderOptions";
 export default class Dropdown extends Component {
   static propTypes = {
     /**
-     * HTML ID attribute
+     * The default value when the component is uncontrolled
      */
-    id: PropTypes.string,
+    defaultValue: PropTypes.oneOfType([
+      PropTypes.any,
+      PropTypes.arrayOf(PropTypes.any)
+    ]),
     /**
-     * Text describing what the field represents
+     * Prevents user actions on the field
      */
-    label: PropTypes.string,
+    disabled: PropTypes.bool,
+    /**
+     * Used to format options into human readable strings
+     *
+     * Note that if both formatOption and renderOption are provided,
+     * renderOption will take precedence
+     */
+    formatOption: PropTypes.func,
```

Comment:
i think it would be good to move `renderOption` up here since you refer to it as well

Reply:
Prefer alpha order, but don't feel super strongly about it. Will update

Reply:
I'm into alphabetical order for prop types.